src/viam/rpc/server.py

```diff
@@ -45,6 +46,21 @@ def __init__(self, components: List[ResourceBase], *, module_service: Optional["
         if module_service is not None:
             services.append(module_service)
         services = ServerReflection.extend(services)
+
+        # Go into every service and replace the methods with a wrapped method that has error handling
+        for service in services:
```

Comment:
we don't actually want to do this - we only want to wrap methods that derive from `ResourceRPCServiceBase`, SignalingService for example we don't need to wrap